frame/identity/src/types.rs

```diff
@@ -333,6 +333,37 @@ pub struct IdentityInfo<FieldLimit: Get<u32>> {
 	pub twitter: Data,
 }
 
+impl<FieldLimit: Get<u32>> IdentityInfo<FieldLimit> {
+	pub(crate) fn fields(&self) -> IdentityFields {
+		let mut res = <BitFlags<IdentityField>>::empty();
+		if self.display != Data::None {
```

Comment:
Some nity nitpick. I would have added a `is_none` method to `Data` and would have called it here everywhere.